guides/release/components/customizing-a-components-element.md

```diff
@@ -33,6 +34,34 @@ export default Component.extend({
 </ul>
 ```
 
+If `tagName` is null:
+
+When the `tagName` is assigned to null value, default `div` tag will be provided by ember but element bindings such as `classNameBindings` will not be available for the root element in component. 
```

Comment:
since we typically avoid putting things in the guides that we don't want users to do, we would want to add some prose here about how this is not a recommended path, but that developers might want to understand what is happening if they see this in some code.

Reply:
@MelSumner : Thanks for your suggestion. Do you think it make more sense to add a new note on this under **Application Concerns** section in the guides?

Reply:
@MelSumner : I modified the file based on your suggestions. Please let me know if you have any more comments/suggestions.

Reply:
@MelSumner : I modified the file based on your suggestions. Please let me know if you have any more comments/suggestions.